Pull Last Updated info from GitHub to accurately reflect last update

mgaudet opened the pull request:
The previous use of site.time would lie because it would only use the time Jekyll ran

hotsphink commented on tech_tree/index.md:
```diff
@@ -100,3 +98,16 @@ stream, as we could then parse incoming chunks off the network.
+<script>
+  // Technique stolen from https://cogitorium.info/2021/02/jekyll-github-revision
+  fetch("https://api.github.com/repos/{{ site.github.repository_nwo }}/commits?path={{ page.path }}&page=1&per_page=2")
+  .then((response) => { return response.json(); })
```
Your aesthetic choices do not match mine and therefore are wrong. `.then(response => response.json())` is prettier.

(Yes, you may ignore this comment, as I am confident in the absolute superiority of my aesthetics and am not insecure enough to need others to adopt them to maintain my self-esteem.)

mgaudet replied:
:) Fixed.

hotsphink commented on tech_tree/index.md:
```diff
@@ -100,3 +98,16 @@ stream, as we could then parse incoming chunks off the network.
+  .then((json) => {
+    if (json.length > 1) {
+      var d = new Date(json[0].commit.author.date);
+      document.getElementById("lastUpdated").innerHTML =  d.toLocaleDateString() + " (" + d.toLocaleTimeString() + ")"
```
`s/innerHTML/innerText/` please. I'm sure there is no locale where `toLocaleDateString` will produce `<iframe src="https://` or whatever, but `.innerHTML` is scary and should not be used lightly.

And if you want nitpicks, then fix the double space and `s/var/const/` or (`let` if you are anti-`const`). But I don't really care about those.

mgaudet replied:
Fixed :)

mgaudet committed:
Pull Last Updated info from GitHub to accurately reflect last update

The previous use of site.time would lie because it would only use the time Jekyll ran
